server/adapters/index.js

```diff
@@ -1,5 +1,7 @@
 module.exports = function initAdapters(libs, conf, models, workers) {
   const adapters = {};
 
+  require('./logAdapter')(models, adapters);
```

Comment:
@michau-k @vuongc @StevenIndigo 
As `logAdapter.js` is in the `adapters` folder, we will not append `Adapter` one more time to it. The same thing is also true for every file in each sub-folder (`adapters`, `controllers`, etc.).